### DISCUSSION TITLE
请教TCGA中特定外显子突变

### DISCUSSION THREAD

**Discussion by zhaoliang0302** [ShixiangWang/self-study#83] [Q&A] [answered]
翔哥好，请问TCGA中可以看到某个基因某个外显子的突变与否吗？
![image](https://user-images.githubusercontent.com/42333702/128127877-bc852111-4803-47bc-8125-9b6164cdd5f6.png)
这是xena下载的MuTect2处理的wes数据，我根据基因外显子去的染色体起止就能确定吗？
谢谢！

**Comment by ShixiangWang** [comment, marked as answer]
我记得 TCGA 下载的 maf 有一列说明了突变发生在哪个外显子上，如果没有确认这个信息的话有2种办法：

1. 下载 gencode 数据库的注释文件，按照你的逻辑将突变位点匹配到外显子上去。
2. 使用突变注释软件，应该是支持注释到exon的。像VEP。

**Reply by zhaoliang0302** [reply]
感谢翔哥，TCGA的突变数据通过TCGAbiolinks下载确实是有这一列的，找到了，谢谢！
